nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Run/ApiModel/Job.cs

```diff
@@ -103,7 +103,7 @@ string CreateIdentifier(string dependencyName, string dependencyVersion)
     public bool IsDependencyIgnoredByNameOnly(string dependencyName)
     {
         var packageNamesToIgnore = IgnoreConditions
-            .Where(c => c.UpdateTypes.Length == 0 && c.VersionRequirement is null) // ignoring by name means there can't be any qualification
+            .Where(c => (c.UpdateTypes ?? []).Length == 0 && c.VersionRequirement is null) // ignoring by name means there can't be any qualification
```

Comment:
The call to `c.UpdateTypes.Length` was throwing a `NullReferenceException` because the `job.json` file looked like this:

``` jsonc
{
  // ...
  "ignore-conditions": [
    {
      "dependency-name": "Some.Dependency",
      "update-types": null // this was the problem
    }
  ]
  // ...
}
```